### PR TITLE
Add announcement for nixos meetup

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: content/articles/london-nixos-meetup-1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Title: First London NixOS meetup
+Date: 2015-03-26
+Short_summary: We are organising a NixOS meetup!
+Category: Events
+Authors: Vincent
+
+Hey everyone,  
+
+We are organising a NixOs meetup in London in [Mozilla offices](https://goo.gl/maps/EyM1r) the 21st of April at 7pm.
```
Comment (teh) [inline content/articles/london-nixos-meetup-1.md]: "the 21st of April at 7pm, in space generously provided by Mozilla London (it's a really cool space!)"

(to say thanks to Mozilla, that's the custom)

File: content/articles/london-nixos-meetup-1.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Short_summary: We are organising a NixOS meetup!
+Category: Events
+Authors: Vincent
+
+Hey everyone,  
+
+We are organising a NixOs meetup in London in [Mozilla offices](https://goo.gl/maps/EyM1r) the 21st of April at 7pm. 
+Tickets are limited and you can grab them on [Eventbrite](https://www.eventbrite.com/e/london-nixos-meetup-tickets-16331038576).  
+
+There will be pizzas, a talk and socializing.
```
Comment (teh) [inline content/articles/london-nixos-meetup-1.md]: "There will be pizza, drinks, a talk and socializing."

File: content/articles/london-nixos-meetup-1.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Category: Events
+Authors: Vincent
+
+Hey everyone,  
+
+We are organising a NixOs meetup in London in [Mozilla offices](https://goo.gl/maps/EyM1r) the 21st of April at 7pm. 
+Tickets are limited and you can grab them on [Eventbrite](https://www.eventbrite.com/e/london-nixos-meetup-tickets-16331038576).  
+
+There will be pizzas, a talk and socializing.  
+Since this is the first event, we will try to figure out what people want out of it.
```
Comment (teh) [inline content/articles/london-nixos-meetup-1.md]: Not sure we need this line... We'll learn that anyway :)

**Comment by teh** [comment]
LGTM after changes, thanks!

File: content/articles/london-nixos-meetup-1.md (hunk 3 above)
Comment (Keats) [inline content/articles/london-nixos-meetup-1.md]: Was just filling the article a bit so the article summary has something to cut x)

**Pull request merged**